modin/experimental/core/execution/native/implementations/omnisci_on_native/io/io.py

```diff
@@ -466,13 +466,17 @@ def _read_csv_check_support(
             )
 
         parse_dates_unsupported = isinstance(parse_dates, dict) or (
-            isinstance(parse_dates, list) and isinstance(parse_dates[0], list)
+            isinstance(parse_dates, list)
+            and any(
+                isinstance(date, list) or not isinstance(date, str)
```

Comment:
Does this mean that for OmniSci to work, all of the values in the list have to be strings?
If so:
```suggestion
                not isinstance(date, str)
```
isn't that equivalent?